README.md

```diff
@@ -23,12 +23,14 @@ TL;DR : With external authz backends like the LDAP one we highly recommend using
 Make sure the connection timeouts in your LDAP server are larger than your configured timeout (`auth_ldap.timeout`) otherwise your LDAP server may terminate the connection and the LDAP plugin may fail to operate afterwards. RabbitMQ 3.7.6 and later versions have addressed these re-connection issues.
 
 ## Implemented Integration scenarios
+
```

Comment:
This new line and the one suggested on line 33 were already incorporated on this commit : #560ac69d777768608f864519e615cb669bbc20fe